docs/docs/v1.0.x/in-repo-addon-plugin.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Creating a plugin as an in-repo-addon
+---
+
+So, you want to prototype a plugin or write one that is so specific to your app that it's not worth making a new npm package for?
+
+Ember CLI allows you to create addons that exist within an existing repository. Here's how to make an ember-cli-deploy as an in-repo-addon.
+
+Generate the plugin
+-------------------
+
+First, use the handy ember-cli generator for creating an in-repo-addon. Don't forget to name the plugin starting with `ember-cli-deploy-`.
```

Comment:
Oh. Good to know. I guess the `Creating a plugin` page also needs to be updated then?